angular/core/components/invitation_form/invitation_form.coffee

```diff
@@ -21,17 +23,33 @@ angular.module('loomioApp').factory 'InvitationForm', ->
       $scope.addCustomMessageClicked = true
 
     $scope.invitees = ->
-      _.compact $scope.form.emails.split(' ')
+      $scope.form.emails.match(/[^\s<,]+?@[^>,\s]+/g) or []
 
     $scope.maxInvitations = ->
       $scope.invitees().length > 100
 
-    $scope.getTeamLink = ->
-      ModalService.open TeamLinkModal, group: -> $scope.form.group()
+    $scope.submitText = ->
+      if $scope.form.emails.length > 0
+        'invitation_form.send'
+      else
+        'invitation_form.done'
+
+    $scope.invalidEmail = ->
+      $scope.invitees().length == 0 and $scope.form.emails.length > 0
 
     $scope.canSubmit = ->
       $scope.invitees().length > 0 and $scope.form.group()
 
+    $scope.invitation = ->
+      group.shareableInvitation()
+
+    $scope.copied = ->
+      FlashService.success('common.copied')
+
+    $scope.invitationLink = ->
+      return unless group.shareableInvitation()
```

Comment:
I think we need to be updating this based on $scope.form.group(), since `group` is just the group passed into the form initially